### DISCUSSION TITLE
Copy & Paster Pictures into Wiki Js

### DISCUSSION THREAD

**Discussion by alexfrehse** [requarks/wiki#7063] [Help / Questions]
Hello,

our current version of wiki js does not allow us to directly copy and paste images into our knowledge base before uploading it. Is it possible to do that in the current version?

Thank you.

**Comment by andymarden** [comment]
Yes - come on, it can't be that difficult for it to paste in and put it in a "pasted" subfolder for assets (or a configurable place). Too many mouse clicks atm.

**Comment by kubilaykaptanoglu** [comment]
Is this topic in your future plans?

**Comment by workdojos** [comment]
Would be nice.  I'm wearing myself out trying to do the simplest of things.
